homeworks/natalia.kokovina_NataliKokovina/homework_5/task3.js

```diff
@@ -0,0 +1,14 @@
+
+const obj = { b: 'c', d: { e: 'f' } };
+
+const objCopy = {};
+function innerCopy(someObj) {
+    Object.keys(someObj).forEach((key) => {
+        objCopy[key] = someObj[key];
```

Comment:
this will not make a deep copy of object, `d` property of both objects still reference the same object

Reply:
changed code